packages/create-resolve-app/bin/index.js

```diff
@@ -3,9 +3,13 @@
 const commandLineArgs = require('command-line-args');
 const moduleCreator = require('../dist/create_resolve_app');
 
-const optionDefinitions = [{ name: 'scripts', type: String }, { name: 'sample', type: Boolean }];
+const optionDefinitions = [
+  { name: 'scripts', type: String },
+  { name: 'sample', type: Boolean },
+  { name: 'version', alias: 'v', type: String }
```

Comment:
Please use 'v' alias in the uppercase, sample code:
 { name: 'version', alias: 'V', type: String }

Reply:
Maybe `version` ia not good name for that purpose, but this `version` is not about CRA version, but about actial NPM published alpha for `resolve-scripts`

Reply:
In current context `version` is like `scripts` field, but accepts NPM version istead of path to target package

Reply:
Also, this PR is only for CI/CD R&D purposes, it will be removed after Jenkins will configured well